.github/workflows/pr.yml

```diff
@@ -0,0 +1,21 @@
+name: Node.js branch push
+on:
+  push:
+    branches:    
+      - '*'         # matches every branch that doesn't contain a '/'
+      - '*/*'       # matches every branch containing a single '/'
+      - '**'        # matches every branch
+      - '!master'   # excludes master
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-node@v2
+        with:
+          node-version: "16.x"
```

Comment:
I would suggest adding a matrix of versions, so we run the tests at least with the latest 3 node LTS versions (14, 16, 18)